src/zodbshootout/fork.py

```diff
@@ -11,7 +11,8 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-"""Multiprocessing utilities.
+"""
```

Comment:
I prefer the [Django convention](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/) which doesn't require an opening sentence to be beside the quotes.